quality_control_stock/models/stock_picking_type.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2014 Serv. Tec. Avanzados - Pedro M. Baeza
+# Copyright 2018 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class StockPickingType(models.Model):
+    _inherit = 'stock.picking.type'
+
+    @api.multi
+    def _create_qc_trigger(self):
+        self.ensure_one()
+        qc_trigger = {
+            'name': self.name,
+            'picking_type_id': self.id,
+            'partner_selectable': True,
+        }
+        result = self.env['qc.trigger'].sudo().create(qc_trigger)
+        return result
+
+    @api.model
+    def create(self, vals):
+        picking_type = super(StockPickingType, self).create(vals)
+        picking_type._create_qc_trigger()
+        return picking_type
+
+    @api.multi
+    def write(self, vals):
+        res = super(StockPickingType, self).write(vals)
+        if vals.get('name') or vals.get('warehouse_id'):
+            qc_trigger_model = self.env['qc.trigger'].sudo()
+            qc_trigger = qc_trigger_model.search(
```

Comment:
Please, could you cherry-pick this commit https://github.com/OCA/manufacture/commit/81e8c99914f8bcb53873ffcf40b3a8815f12e1de of v10?

Reply:
Done!